compiler/codegen/src/compile.rs

```diff
@@ -2155,10 +2158,7 @@ impl Compiler<'_> {
             for ident in attrs.iter().take(n_attrs).skip(i + 1) {
```

Comment:
I know this isn't the code you changed, but if you're looking to make it more rusty, you could use `.enumerate()` here for the outer loop (`for (i, attr) in attrs.iter().enumerate()`) and then just `for ident in &attrs[i + 1..]` for the inner loop.